include/mvd/bits/mvd3_misc.hpp

```diff
@@ -90,18 +93,24 @@ const std::string did_cells_layer = "/cells/properties/layer";
 const std::string did_cells_index_morpho = "/cells/properties/morphology";
 const std::string did_cells_index_etypes = "/cells/properties/etype";
 const std::string did_cells_index_mtypes = "/cells/properties/mtype";
+const std::string did_cells_index_emodel = "/cells/properties/me_combo";
```

Comment:
I think it would be better to rename `emodel` to `me_combo` to avoid any confusion with `emodel` of the `tsv` file, since those are different things as far as I understood.
```suggestion
const std::string did_cells_index_me_combo = "/cells/properties/me_combo";
```
See also the other suggestions

Reply:
Until yesterday I thought the me_combo was the name of the file to be loaded. Some conversations induced me into error.
Having that said, true, we can have this field in mvd3, but we still must implement the function in `getEmodels()` in mvd3 otherwise the class is abstract.
Since that field only comes from the TSV, I can implement it with a throw() inside. Then It's up to you to modify it and return something in case the mvd is loaded :)

Reply:
Exactly, the `getEmodels()` function for the `MVD3` file in my mind should be like this:
```
inline std::vector<std::string> MVD3File::getEmodels(const Range& range) const{
    if (!_tsv_file) {
        std::ostringstream ss;
        ss << "No tsv file is opened with MVD3 file " << _filename << " to extract eModel information from"
           << std::endl;
        throw MVDException(ss.str());
    }
    std::vector<std::string> morphologies = getMorphologies(range);
    std::vector<std::string> me_combos = getMECombos(range);
    return _tsv_file->getEmodels(me_combos, morphologies);
}
```
(The code comes from here https://github.com/BlueBrain/MVDTool/pull/65)